plugins/brimcap/brimcap-plugin.ts

```diff
@@ -171,27 +169,24 @@ export default class BrimcapPlugin {
     )
   }
 
-  private logToSearchOpts(log: zng.Record): searchOptions {
-    const ts = log.get("ts") as zng.Primitive
+  private logToSearchOpts(log: zed.Record): searchOptions {
+    const ts = log.get("ts") as zed.Time
     // RFC3999nano format with zero timezone offset
-    const formatter = DateTimeFormatter.ofPattern(
-      "yyyy-MM-dd'T'HH:mm:ss.SSSSSSSSS'Z'"
+    // const formatter = DateTimeFormatter.ofPattern(
+    //   "yyyy-MM-dd'T'HH:mm:ss.SSSSSSSSS'Z'"
+    // )
+
+    const tsString = ts.toString()
```

Comment:
Will this still preserve nano second precision? I threw in this js-joda stuff here to help with the formatting since `brimcap` explicitly expects RFC3999nano

Reply:
Yes, the time values from zjson are now represent as RFC39999nano strings. I moved the js-joda code in the zed.Time class because I need to parse them there.